### PR TITLE
Add array_flip term entry

### PR DESCRIPTION
**Pull request by braanj**
### Description

Add a new term entry: PHP flip_array()

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/array-flip/array-flip.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  - 'Web Design'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_flip()`** method is used to interchange the keys and values of an array. It creates a new array where the original array's values become the keys, and the original array's keys become the values. This function can be useful in situations where you need to quickly lookup values based on their previous keys.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-flip/array-flip.md:15]:
```suggestion
The **`array_flip()`** method is used to interchange the keys and values of an array. It creates a new array where the original array's values become the keys, and the keys become the values. This function can be useful in situations where you need to quickly lookup values based on their previous keys.
```

File: content/php/concepts/arrays/terms/array-flip/array-flip.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+```pseudo
+array_flip($original_array)
+```
+
+The `array_flip()` function has one required parameter:
+
+- `$original_array`: The array to be flipped.
+
+The `array_flip()` function returns a flipped array.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-flip/array-flip.md:27]:
```suggestion
```

File: content/php/concepts/arrays/terms/array-flip/array-flip.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+## Example
+
+The following example uses the `array_flip()` function to flip an ``$original_array` and store the resulted array on `$flipped_array`:
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-flip/array-flip.md:31]:
```suggestion
The following example uses the `array_flip()` function to flip `$original_array` :
```

File: content/php/concepts/arrays/terms/array-flip/array-flip.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+The example will result in the following output:
+
+```shell
+Array (
+  ['fruit'] => 'apple',
+  ['vegetable'] => 'carrot'
+)
+```
+
+## Codebyte Example
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-flip/array-flip.md:55]: Examples within a term entry should not be duplicative. Each example should ideally demonstrate a different aspect of the functionality, new information, etc. Please revise this example or delete it.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@braanj I've added my comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by braanj** [comment]
@caupolicandiaz Changes made!

**Review by braanj — COMMENTED** [review]
"The `array_flip()` function returns a flipped array." was removed

File: content/php/concepts/arrays/terms/array-flip/array-flip.md (hunk 4 above)
Comment (braanj) [reply content/php/concepts/arrays/terms/array-flip/array-flip.md:53]: The example is changed

**Review by caupolicandiaz — APPROVED** [review]
@braanj Looks good. I'm sending this along for a second review.

File: content/php/concepts/arrays/terms/array-flip/array-flip.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  - 'Web Design'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_flip()`** method is used to interchange the keys and values of an array. It creates a new array where the original array's values become the keys, and the keys become the values. This function can be useful in situations where you need to quickly lookup values based on their previous keys.
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-flip/array-flip.md:15]:
```suggestion
The **`array_flip()`** method is used to interchange the keys and values of an array. It creates a new array where the original array's values become the keys, and the keys become the values. This function can be useful in situations where it's needed to quickly lookup keys based on their previous values.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@braanj I did a review and left a comment.

**Comment by braanj** [comment]
@SSwiniarski Thanks for the review

**Review by SSwiniarski — APPROVED** [review]
@braanj Looks good.

**Comment by SSwiniarski** [comment]
@braanj Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/php/arrays/array-flip

image

**Pull request merged**